internal/gapicgen/generator/config.go

```diff
@@ -14,6 +14,26 @@
 
 package generator
 
+import "strings"
+
+// gapicPkgs is a map of googleapis inputDirectoryPath to the gapic package name
+// used for conventional commits.
+var gapicPkgs map[string]string
+
+func init() {
+	gapicPkgs = make(map[string]string)
+	for _, v := range microgenGapicConfigs {
+		gapicPkgs[v.inputDirectoryPath] = parseConventionalCommitPkg(v.importPath)
+	}
+}
```

Comment:
It might be more clear, though possibly less efficient, to just call `parseConventionalCommitPkg` when the value is needed, rather than precomputing `gapicPkgs`. Removes a layer of indirection.

Reply:
Yeah using init never really feels great. I think this map is only used in one spot. Maybe I will just build it in place.